gobcore/events/import_message.py

```diff
@@ -0,0 +1,92 @@
+"""GOB Header
+
+The GOB Header holds the header values of a workflow message.
+
+Todo: Name of the module (message) should correspond to the contents of the file (GOBHeader)
+
+"""
+from gobcore.exceptions import GOBException
+
+
+class MessageMetaData():
+
+    source_id_column = '_source_id'
+
+    def __init__(self, header):
+        self._header = header
+
+    @property
+    def source(self):
+        return self._header['source']
+
+    @property
+    def timestamp(self):
+        return self._header['timestamp']
+
+    @property
+    def id_column(self):
+        return self._header['entity_id']
+
+    @property
+    def entity(self):
+        return self._header['entity']
+
+    @property
+    def version(self):
+        return self._header['version']
+
+    @property
+    def model(self):
+        return self._header['gob_model']
+
+    @property
+    def as_header(self):
+        return {
+            "source": self.source,
+            "timestamp": self.timestamp,
+            "entity_id": self.id_column,
+            "entity": self.entity,
+            "version": self.version,
+            "gob_model": self.model
+        }
+
+
+class ImportMessage():
+
+    def __init__(self, msg):
+        self._metadata = MessageMetaData(msg["header"])
+
+        if self.metadata.source is None or \
+                self.metadata.source is None or \
```

Comment:
test on None source is duplicated
Probably use
[ self.metadata[attr] for attr in ['source', 'timestamp' ... ]].count(None) > 0

Reply:
Removed the duplicate. Thanks. I think I prefer the readability of this version